lib/pages/chat/events/message/message_style.dart

```diff
@@ -33,7 +33,10 @@ class MessageStyle {
         : const Color.fromARGB(239, 36, 36, 36);
   }
 
-  static int get messageFlexMobile => 7;
+  static int messageFlexMobile(bool othersMessageNotInGroupChat) {
+    return othersMessageNotInGroupChat ? 3 : 5;
```

Comment:
IMO: With your rate quite small for this monitor

Reply:
What about in landscape mode in mobile

Reply:
IMO it's fine like this doesn't take much space in landscape mode
I did 3 for others message in group chat because there is avatar 
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/bc1ae10c-dc9f-4363-8730-ba8c07fe49ad)
![demo_msg_width222](https://github.com/linagora/twake-on-matrix/assets/48354990/5e680403-c996-4899-bd85-916b03dc4d32)

Reply:
IMO: It's okay

Reply:
hope it bigger in portrait screen. ( I talked in details with Julian in meeting)

Reply:
![demodemo](https://github.com/linagora/twake-on-matrix/assets/48354990/a2ad66ea-0290-4cab-8388-d76ac4b58bbc)
this is 4 instead of 3 flex

Reply:
IMO, it is still a lot of space in view. Try to keep the UI good but need to prioritize the main function of communication.
We need to align the left/right and also manage the maximum width. WDYT?

Reply:
for me we should put a fixed margin and update  it if the device is mobile or web

Reply:
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/8130f3a9-a41b-4dc9-a40b-87a908e2becf)
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/50589578-3fe0-4b5e-9d82-8a675618b2a5)

Reply:
how about demo in web or landscape?

Reply:
web version doesn't use this style, it looks like before 
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/77a4da21-ad21-4b2d-90bd-a1759745111f)